tools/clang/unittests/HLSL/PixTest.cpp

```diff
@@ -820,8 +1073,9 @@ static bool FindStructMemberFromStore(llvm::StoreInst *S,
   return false;
 }
 
-// This function lives in lib\DxilPIXPasses\DxilAnnotateWithVirtualRegister.cpp
-// Declared here so we can test it.
+// This function lives in
+// lib\DxilPIXPasses\DxilAnnotateWithVirtualRegister.cpp Declared here so we
+// can test it.
```

Comment:
These don't look like what clang-format would have done, since they were previously 79 characters long, so why the changes here?

Reply:
(Probably from the clang-format setting in VS being global, and me switching back and forth between dxc and PIX and forgetting to toggle the setting.)